### PR TITLE
[Concept Entry] Sklearn multioutput-regression

### LINKED ISSUE #5000: [Concept Entry] Sklearn multioutput-regression
### Reason/inspiration (optional)

We would like a new entry on the `Multioutput Regression` concept under Sklearn. The entry should go in a new file under `docs/content/sklearn/concepts/multioutput-regression/multioutput-regression.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept (if applicable)
- An `Example` section that provides an example demonstrating the concept in use (if applicable)
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use (if applicable)

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by mamtawardhani**
### Description
Added Concept Entry for Sklearn multioutput-regression

### Issue Solved
Closes #5000 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/sklearn/concepts/multioutput-regression/multioutput-regression.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  - 'Classification'
+  - 'Multitask Learning'
+  - 'MultiTaskLasso'
+  - 'Scikit-learn'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+In sklearn, **Multioutput Regression** is a type of regression task where the model predicts multiple dependent variables (outputs) simultaneously for each input, allowing for the modeling of relationships between multiple target variables and the features, which can improve prediction accuracy when outputs are correlated.
```
Comment (PragatiVerma18) [inline content/sklearn/concepts/multioutput-regression/multioutput-regression.md:17]: Add a link to sklearn docs entry.

```suggestion
In [sklearn](https://www.codecademy.com/resources/docs/sklearn), **Multioutput Regression** is a type of regression task where the model predicts multiple dependent variables (outputs) simultaneously for each input, allowing for the modeling of relationships between multiple target variables and the features, which can improve prediction accuracy when outputs are correlated.
```

File: content/sklearn/concepts/multioutput-regression/multioutput-regression.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+This can be achieved using the `MultiOutputRegressor` class, which wraps a single-output regressor (like `LinearRegression` or `DecisionTreeRegressor`) and fits a separate model for each target variable. The model then predicts all outputs at once for each input, treating each target independently.
```
Comment (PragatiVerma18) [inline content/sklearn/concepts/multioutput-regression/multioutput-regression.md:19]: We can add links to Linear Regression and Decision Tree Regressor here as well:

```suggestion
This can be achieved using the `MultiOutputRegressor` class, which wraps a single-output regressor (like [`LinearRegression`](https://www.codecademy.com/resources/docs/sklearn/linear-regression-analysis) or [`DecisionTreeRegressor`](https://www.codecademy.com/resources/docs/sklearn/decision-trees)) and fits a separate model for each target variable. The model then predicts all outputs at once for each input, treating each target independently.
```

File: content/sklearn/concepts/multioutput-regression/multioutput-regression.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+## Syntax
+
+```pseudo
+sklearn.multioutput.MultiOutputRegressor(estimator, n_jobs=None)
```
Comment (PragatiVerma18) [inline content/sklearn/concepts/multioutput-regression/multioutput-regression.md:24]: In Scikit-learn, `MultiOutputRegressor` is located in the `sklearn.multioutput` module. However, the `sklearn.multioutput` part is just the module path, and `MultiOutputRegressor` needs to be imported explicitly from that module. It can't be used as in current syntax.

So, the correct way to use it would be:

```suggestion
from sklearn.multioutput import MultiOutputRegressor

multi_output_regressor = MultiOutputRegressor(estimator, n_jobs=None)
```

File: content/sklearn/concepts/multioutput-regression/multioutput-regression.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+- `estimator`: The base regressor that is used to fit each target independently. This can be any regression model that supports single-output regression (e.g., `LinearRegression`, `DecisionTreeRegressor`, etc.).
+- `n_jobs`: The number of jobs to run in parallel for fitting the models.
+  - If `None`, it defaults to 1 (single-threaded).
+  - If `-1`, it uses all available processors.
+  - If `int > 1`, it uses that many processors for parallel computation.
+
+## Example
+
+In the following example, a multi-output regression model is trained using `MultiOutputRegressor` with `LinearRegression` as the base estimator to predict two target variables from a dataset with 100 samples and 10 features:
```
Comment (PragatiVerma18) [inline content/sklearn/concepts/multioutput-regression/multioutput-regression.md:35]:
```suggestion
In the following example, a multi-output regression model is trained using `MultiOutputRegressor` with `LinearRegression` as the base `estimator` to predict two target variables from a dataset with 100 samples and 10 features:
```

File: content/sklearn/concepts/multioutput-regression/multioutput-regression.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+ [ -10.61243193  289.0884239 ]
+ [  73.11234969  158.84947994]
+ [  10.45019796  260.51876186]
+ [-226.04884764 -372.71451196]
+ [ -17.30979575 -146.3735002 ]
+ [ -13.07113033  -42.21748842]
+ [ -59.54942557 -102.03957313]]
+```
+
+> **Note:** The output will vary each time the code is run unless a fixed `random_state` is set in `make_regression()` to ensure reproducibility.
```
Comment (PragatiVerma18) [inline content/sklearn/concepts/multioutput-regression/multioutput-regression.md:164]: The phrasing can be made slightly clearer regarding the role of `random_state`:

```suggestion
> **Note:** The output will vary each time the code is run unless a fixed `random_state` is set in `make_regression()`, ensuring reproducibility as shown in the example.
```

**Review by PragatiVerma18 — CHANGES_REQUESTED** [review]
Hey @mamtawardhani, thanks for adding this entry. I have some suggestions that I have added as comments. Please check and do the requested changes. Let me know if you have any questions. Happy contributing! 🥳

**Comment by mamtawardhani** [comment]
Hi @PragatiVerma18, thanks for the feedback, the changes are made 😄

**Review by PragatiVerma18 — APPROVED** [review]
Hey @mamtawardhani, thanks for doing the changes so quickly. This looks good to be merged now! 🚀
